doc/source/user_guide/advanced.rst

```diff
@@ -965,6 +965,28 @@ If you select a label *contained* within an interval, this will also select the
    df.loc[2.5]
    df.loc[[2.5, 3.5]]
 
+Selecting using an ``Interval`` will only return exact matches.
```

Comment:
I would maybe put this (together we the above examples) in a subsection on indexing with an IntervalIndex

Reply:
Created a section for indexing, and also one for binning since the `cut` examples seemed out of place otherwise.